fastlane-plugin-wpmreleasetoolkit.gemspec

```diff
@@ -43,8 +43,13 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'progress_bar', '~> 1.3'
   spec.add_dependency 'parallel', '~> 1.14'
   spec.add_dependency 'chroma', '0.2.0'
-  spec.add_dependency 'activesupport', '~> 6.0'
-  spec.add_dependency 'bigdecimal', '~> 2.0'
+  spec.add_dependency 'activesupport', '~> 5'
+  # Some of the upstream code uses `BigDecimal.new` which version 2.0 of the
+  # `bigdecimal` gem removed. Until we'll find the time to identify the
+  # dependencies and see if we can move them to something compatible with
+  # modern `bigdecimal`, let's constrain the gem to a version still supporting
+  # `.new` but which warns about it deprecation.
+  spec.add_dependency 'bigdecimal', '~> 1.4'
```

Comment:
I agree with the rationale of reverting our change since it's breaking. But I'm wondering if we couldn't just remove those dependencies from release toolkit completely – tbh I'm not really sure we use activesupport and bigdecimal directly from within the toolkit? If we do it would be worth identifying where and why.

I think the right course of action will be to:
 - Approve and ship that PR that reverts the changes
 - Ship a new version of the release toolkit to include our other PRs (hotfix lanes, skip deliverfile)
 - Revisit that dependency removal after that, by doing a quick investigation on where it's used in the RT and why

Reply:
> But I'm wondering if we couldn't just remove those dependencies from release toolkit completely

That would be idea. Best way to solve a problem is to not have the problem in the first place!